test/e2e/scenario/telemetry.scenario.ts

```diff
@@ -37,11 +36,41 @@ describe('telemetry', () => {
         window.DD_RUM!.addAction('hop', context as any)
       })
       await flushEvents()
-      expect(serverEvents.telemetry.length).toBe(1)
-      const event = serverEvents.telemetry[0] as TelemetryErrorEvent
+      expect(serverEvents.telemetryErrors.length).toBe(1)
+      const event = serverEvents.telemetryErrors[0]
       expect(event.telemetry.message).toBe('bar')
       expect(event.telemetry.error!.kind).toBe('Error')
       expect(event.telemetry.status).toBe('error')
       serverEvents.empty()
     })
+
+  createTest('send init configuration for logs')
+    .withSetup(bundleSetup)
+    .withLogs({
+      enableExperimentalFeatures: ['telemetry_configuration'],
+      forwardErrorsToLogs: true,
+    })
+    .run(async ({ serverEvents }) => {
+      await flushEvents()
+      expect(serverEvents.telemetryConfigurations.length).toBe(1)
+      const event = serverEvents.telemetryConfigurations[0]
+      expect(event.telemetry.type).toBe('configuration')
+      expect(event.telemetry.configuration.forward_errors_to_logs).toEqual(true)
+      serverEvents.empty()
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Do we need those `serverEvents.empty()`?

Reply:
I updated the `tearDownTest` to only test that we should not have telemetry errors so I can remove `serverEvents.empty()`
https://github.com/DataDog/browser-sdk/blob/f22ccfcc69c9ac16466a778e49dc299252514160/test/e2e/lib/framework/createTest.ts#L193-L195

Reply:
ok got it!